src/inferencetensor.hpp

```diff
@@ -17,21 +17,22 @@
 #include <memory>
 #include <optional>
 #include <string>
+#include <vector>
 
-#include "ovms.h"  // NOLINT
-#include "shape.hpp"
+#include "ovms.h"     // NOLINT
+#include "shape.hpp"  // TODO: remove?
```

Comment:
TODO to be removed since we need "using"?

Reply:
Removing todo, shape include needs to stay since `signed_shape_t` definition is in there